WindowsMRAssetConverter/CommandLine.h

```diff
@@ -8,12 +8,19 @@
 
 namespace CommandLine
 {
+    enum Platform
+    {
+        None = 0x0,
+        Holographic = 0x1,
+        Desktop = 0x2
+    };

```

Comment:
The interface of the library uses the packing types. This is a lot more user-friendly, so that the dev doesn't have to study the docs to figure out which packing to use. 
Here I'm using `-compatibility-mode <on|off>` and `-platform <holographic|desktop|all>` in combination to get assets with the right packing (NRM or ORM, but also supporting NRM+ORM, ORM+RMO and NRM+RMO+ORM combos).
What would be your proposal?

Reply:
Also provide a option for packing types to add if the customer wants to define this themselves for use outside of Windows MR.

Simplifying with platform is great but a target minversion would be better than compatibility as our requirements/options evolve over time. Even better maybe to parse an appx manifest.

Reply:
If the user wants to use their asset outside of Windows MR then this is the wrong tool. It's very easy to use the library and pack up a tool for your own scenario, but this tool specifically doesn't allow you to choose the packing/compression parameters besides what's supported for WMR.

I thought about the min version but that also adds complexity. I was trying to avoid as much as possible the user of the tool having to learn implementation details in order to use it. I agree that compatibility-mode is a very generic way of saying "support RS3 as well". I guess I'd be OK with changing that parameter to "-min-version <number>" and number could be a build number (e.g. 16299.0) or a version number (1803) and it would do the right thing on either case.

Reply:
Fixed